<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaySoundForm</class>
 <widget class="QWidget" name="PlaySoundForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放</string>
  </property>
  <widget class="QPushButton" name="preVoiceButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>上一句</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playVoiceButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseVoiceButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextVoiceButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下一句</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repeateVoiceButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重复</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
